### PR TITLE
Documentation of functions

### PR DESCRIPTION
**Pull request by choyiny**
Resolving #2 

Would like to continue working on this expanding to more files after the python file merges

### CODE REVIEW

**Comment by the-zebulan** [comment]
@choyiny ,

I get this PEP8 warning about your docstrings in PyCharm:

> Triple double-quoted strings should be used for docstrings.

In other words, just change `'''` -> `"""`.

**Comment by choyiny** [comment]
I was trying to make a docstring, but the format is different to the one of PyCharm. But good point I actually haven't put the code to check for PEP8 compliance... let me do it now.

**Comment by the-zebulan** [comment]
There are a bunch of PEP8 warnings on some of the other files too. I was just going to wait until this pull request was merged (#12) to fix them.

File: main.py
Comment (the-zebulan) [inline main.py:12]: Spelling mistake `coffebot` -> `coffeebot`

File: main.py
Comment (the-zebulan) [inline main.py:27]: PEP8: Triple double-quoted strings should be used for docstrings

File: main.py
Comment (the-zebulan) [inline main.py:14]: Some variable names use `mixedCase` and some use `snake_case`. Should all be `snake_case` as per Python PEP8 style conventions.

https://www.python.org/dev/peps/pep-0008/#naming-conventions

File: main.py
Comment (the-zebulan) [inline main.py:45]: PEP8: Triple double-quoted strings should be used for docstrings

File: main.py
Comment (the-zebulan) [inline main.py:51]: Multiple string concatenations isn't really best practice. You could use something like this instead:

`u'@{} {} {} {}'.format(username, a, order(), b)`

**Review by the-zebulan — CHANGES_REQUESTED** [review]
Here are some of the warnings that are still present using PyCharm.

**Comment by DeastinY** [comment]
https://github.com/DeastinY/CoffeeRecommendationBot/pull/12 is merged. Sorry for sleeping at night :sweat_smile: 
Great initial fixup @choyiny and thanks for the very thorough PEP-8 review @the-zebulan ! As you already pointed them all out, feel free to fix them ! (I think you can self-assign [this issue](https://github.com/DeastinY/CoffeeRecommendationBot/issues/16))

**Pull request merged**
